src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -166,6 +169,12 @@ protected OptionParser buildOptionParser() {
 
 		parser.accepts( "__separator_8" );
 
+		parser.accepts( "metrics_reporting_type", "how maxwell metrics will be reported, at least one of slf4j|jmx|http" ).withOptionalArg();
+		parser.accepts( "metrics_reporting_interval", "the frequency metrics are emitted to the log, in seconds, when slf4j reporting is configured" ).withOptionalArg();
+		parser.accepts( "metrics_reporting_port", "the port the server will bind to when http reporting is configured" ).withOptionalArg();
```

Comment:
i think it's ok. none are actually required (when nothing's supplied no metrics are output anywhere).